api/scripts/localstack-init.php

```diff
@@ -37,73 +38,80 @@ function isLocalstackAvailable()
     'Name' => '/default/flag/document-sync',
     'Type' => 'String',
     'Value' => '1',
-    'Overwrite' => true
+    'Overwrite' => true,
 ]);
 
 $ssmClient->putParameter([
     'Name' => '/default/parameter/document-sync-row-limit',
     'Type' => 'String',
     'Value' => '100',
-    'Overwrite' => true
+    'Overwrite' => true,
 ]);
 
 $ssmClient->putParameter([
     'Name' => '/default/parameter/document-sync-interval-minutes',
     'Type' => 'String',
     'Value' => '4.5',
-    'Overwrite' => true
+    'Overwrite' => true,
 ]);
 
 $ssmClient->putParameter([
     'Name' => '/default/flag/checklist-sync',
     'Type' => 'String',
     'Value' => '1',
-    'Overwrite' => true
+    'Overwrite' => true,
 ]);
 
 $ssmClient->putParameter([
     'Name' => '/default/parameter/checklist-sync-row-limit',
     'Type' => 'String',
     'Value' => '100',
-    'Overwrite' => true
+    'Overwrite' => true,
+]);
+
+$ssmClient->putParameter([
+     'Name' => '/default/parameter/benefits-questions',
+     'Type' => 'String',
+     'Value' => '31-12-2030 00:00:00',
+     'Overwrite' => true,
```

Comment:
Do we need this in API if the feature flag is only on the client side?

Reply:
Good question.

I copied the pattern for the other flags and added it to both scripts. 
However since we're not touching the API side here, it's not really needed in this PR.

I'm not sure if the feature flag will be required on the API side of the app, so I can remove it for now and we can add it back in when it's needed.

Reply:
Yeah, I think if we're not using it yet we should drop it out. If we never need it on the API side, it may be confusing for future selves/new devs.